packages/web/integrations/bridges/skip/types.ts

```diff
@@ -0,0 +1,194 @@
+export type Asset = {
```

Comment:
Can you rename this to `SkipAsset`? I'd like to avoid asset type collision between Osmosis and Skip